packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -110,6 +110,7 @@ export class BundlingDockerImage {
 
     const dockerArgs: string[] = [
       'build', '-q',
+      ...(options.file ? ['-f', options.file] : []),
```

Comment:
I vaguely recall this is already supported. Did you rebase?

Reply:
@eladb I wiped out my previous branch and started fresh from the master branch today - the old commits were not practical to revert. I used the force to push it over the branch in this PR. I didn't see pre-existing support for this [in master](https://github.com/aws/aws-cdk/blob/451200e3dba6b8d80bc8396dc0b7d4d84f29cb56/packages/%40aws-cdk/core/lib/bundling.ts#L111-L115). However, I've got practically the same code in PR #9582, which I believe you've previously reviewed.

Reply:
It's currently supported in `cdk-assets`.

Reply:
The API of the `Docker` class in `cdk-assets` looks pretty handy to use. But, both `core` and `cdk-assets` seem to have minimal dependencies. Any ideas what I should do here?

Reply:
Nothing at this stage, I was just mentioning from where @eladb might have thought that this was already supported.

Not for this PR but at some point it might be interesting to correctly define the "roles" of `core` vs `cdk-assets`. I think that this was already discussed in another issue about where zipping should occur.